app/controllers/sms/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Sms::UsersController < Sms::BaseController
+  def update
+    user = User.find_by_phone(params["From"])
+    if (params["Body"].upcase[0..1] == "NO" || params["Body"].upcase[0..10] == "UNSUBSCRIBE") && user
+      user.active = false
+      user.save
```

Comment:
we could probably move this logic out of the controller into a method called deactivate maybe?